libs/island-ui/contentful/src/lib/BackgroundImage/BackgroundImage.tsx

```diff
@@ -45,7 +45,7 @@ export const BackgroundImage: FC<
     borderRadius: 'large',
   },
 }) => {
-  const src = `${image.url}?w=${width}`
+  const src = `${image?.url}?w=${width}`
```

Comment:
What if image or url are undefined?

Reply:
Yeah. Should there be a fallback or some defaults then?

Reply:
Maybe something like this: 
![image](https://github.com/island-is/island.is/assets/43557895/aef92c01-b781-4c09-9d45-4c54285963ed)